verif/core_rrv/tb/core_rrv_verif_list.f

```diff
@@ -2,3 +2,4 @@
 ../../../verif/rv32i_ref/tb/rv32i_ref_pkg.sv
 ../../../verif/rv32i_ref/tb/rv32i_ref.sv
 ../../../verif/core_rrv/tb/core_rrv_no_ref_tb.sv
+../../../verif/core_rrv/tb/core_rrv_tb.sv
```

Comment:
Always compile both files! 
Decide wich one to use in elaboration & simulation by adding the "-top "
(By default without -top we are using the "core_rrv_tb")